### PR TITLE
fix the install script to download other architectures then x86_64

### PR DESCRIPTION
**Pull request by namtzigla**
the install script had an amd64 hardcoded

### CODE REVIEW

File: tools/misc/get_plz.sh [hunk 1]
```diff
@@ -16,7 +16,17 @@ else
     exit 1
 fi
 
-PLEASE_URL="https://get.please.build/${GOOS}_amd64/${VERSION}/please_${VERSION}.tar.gz"
+ARCH=`uname -m`
+if [ "$ARCH" = "x86_64" ]; then 
+    ARCH="amd64"
+elif [ "$ARCH" = "arm64" ]; then
+    ARCH="amd64"
```
Comment (peterebden) [inline tools/misc/get_plz.sh:23]: Should be `arm64`, right?

**Review by peterebden — COMMENTED** [review]
Thanks - not surprised there are some of these scattered around (back when this was written ARM wasn't very relevant to us... how times change!)

File: tools/misc/get_plz.sh (hunk 1 above)
Comment (namtzigla) [reply tools/misc/get_plz.sh]: yep

**Comment by namtzigla** [comment]
@peterebden @Tatskaari I think is this ready

**Review by peterebden — APPROVED** [review]
Thanks!

**Pull request merged**